server/src/modules/user_Instruments/user_Instruments.controller.ts

```diff
@@ -31,7 +31,10 @@ export class User_InstrumentsController {
   findOne(@Param('id') id: string) {
     return this.user_InstrumentsService.findOne(+id);
   }
-
+  @Get('user/:userId')
```

Comment:
I think this @get should be in the instruments controller (following what Andrea did with user_ensemble). We can move the instruments folder out of the seeder so it's available

Reply:
i think the naming is a bit confusing. From the naming looks like we are retrieving a user, while we are retrieving a user in the user-instrument context. Maybe we can have an approch that could be: 1)/user-instruments/:userId; 2)/user-instruments/user/:userId.

what do you think?



I would keep the logic inside the user_instrument because we are retrieving a user_instrument row. 

@amorea9 what are you referring to ? i cannot remember what i did 😆